frontend/src/store/store.js

```diff
@@ -68,6 +68,12 @@ export const store = new Vuex.Store({
     description: '',
     token: null,
     newsletterState: null,
+    community: {
+      name: 'Gradido Entwicklung',
+      url: 'http://localhost:3000/vue/',
+      registerUrl: 'http://localhost:3000/vue/register',
+      description: 'Die lokale Entwicklungsumgebung von Gradido.',
+    },
```

Comment:
Would be nice to get this out of the .env file

Reply:
i need a little help ...

Reply:
Server's `.env` file